code/modules/surgery/rig_removal.dm

```diff
@@ -0,0 +1,58 @@
+//Procedures in this file: Unsealing a Rig.
```

Comment:
This was intentionally not ported. Player rigsuits shouldn't require surgery to get off (especially because you can't force surgery on someone without knocking them out)